tests/util/helper/cmd/main.go

```diff
@@ -0,0 +1,117 @@
+// Copyright 2018 Istio Authors
```

Comment:
The package for this file seems strange. Should this be under something like `/tests/util/registry/cmd`?

Reply:
well I moved it because I think at some point you'll have more than one thing you want to do to setup your dev env, not just the registry. And since we asking the User to ctrl-c, it makes more sense to have a multi features command.

Reply:
So to reformulate this, I think we want to create a CLI with all the cool stuff from the framework to ease developer set up test system for manual testing.

Reply:
Maybe just treat them as separate binaries for now? I'm not sure I see the utility in globing multiple commands together.  But we can re-evaluate once we have multiple commands.